### PR TITLE
添加鸿蒙平台适配

### PR DESCRIPTION
**Pull request by IceBBBBB**
添加鸿蒙平台适配

### CODE REVIEW

File: cocos/scripting/lua-bindings/script/cocos2d/Cocos2dConstants.lua [hunk 1]
```diff
@@ -182,6 +182,7 @@ cc.PLATFORM_OS_MAC     = 2
 cc.PLATFORM_OS_ANDROID = 3
 cc.PLATFORM_OS_IPHONE  = 4
 cc.PLATFORM_OS_IPAD    = 5
+cc.PLATFORM_OS_OHOS     = 6
```
Comment (minggo) [inline cocos/scripting/lua-bindings/script/cocos2d/Cocos2dConstants.lua]: The value is conflict with `cc.PLATFORM_OS_BLACKBERRY`. It is better to use another number.

File: cocos/scripting/lua-bindings/script/framework/device.lua [hunk 2]
```diff
@@ -58,6 +58,8 @@ elseif target == cc.PLATFORM_OS_WINRT then
     device.platform = "winrt"
 elseif target == cc.PLATFORM_OS_WP8 then
     device.platform = "wp8"
+elseif target == cc.PLATFORM_OS_OHOS then
+    device.platform = "ohos"
```
Comment (minggo) [inline cocos/scripting/lua-bindings/script/framework/device.lua]: Should change to `HarmonyOS Next`?

File: cocos/scripting/lua-bindings/script/cocos2d/Cocos2dConstants.lua (hunk 1 above)
Comment (IceBBBBB) [reply cocos/scripting/lua-bindings/script/cocos2d/Cocos2dConstants.lua]: modified

File: cocos/scripting/lua-bindings/script/framework/device.lua (hunk 2 above)
Comment (IceBBBBB) [reply cocos/scripting/lua-bindings/script/framework/device.lua]: modified

**Pull request merged**